stable/mlrun/templates/api-deployment.yaml

```diff
@@ -60,6 +60,10 @@ spec:
                 name: {{ .Release.Name }}-v3io-fuse
                 key: username
           {{- end }}
+          {{- if .Values.nuclioAPIURL }}
+          - name: MLRUN_NUCLIO_DASHBOARD_URL
```

Comment:
```suggestion
          - name: MLRUN_NUCLIO_API_URL
```

Reply:
This requires a change in MLRun code which I prefer not doing right now, the env var is already `MLRUN_NUCLIO_DASHBOARD_URL`

Reply:
I understand, the issue is that nuclio dashboard now encompasses both api url and ui url, so it's less specific than either